it/v2/connectors/PersistenceConnectorSpec.scala

```diff
@@ -1444,7 +1443,7 @@ class PersistenceConnectorSpec
 
         implicit val hc = HeaderCarrier()
         val r = persistenceConnector
-          .getArrivalMessage(eori, arrivalId, messageId)
+          .getMessage(eori, MovementType.Arrival, arrivalId, messageId)
```

Comment:
I am not sure if I am right here. Can we not combine arrival/departure using forEach with movementType list ? so that we can reduce test cases code as well